src/main.rs

```diff
@@ -240,14 +270,31 @@ pub fn spawn_bananas_over_time(
 
         let random_x = (random::<f32>() * bounds_width) + BOUND_SIZE;
 
-        commands.spawn((
-            SpriteBundle {
-                transform: Transform::from_xyz(random_x, window.height(), 0.0),
-                texture: asset_server.load("sprites/banana.png"),
-                ..default()
-            },
-            Banana {},
-        ));
+        let is_bunch = random::<f32>() < 0.1;
+
+        if is_bunch {
+            commands
+                .spawn((
+                    SpriteBundle {
+                        transform: Transform::from_xyz(random_x, window.height(), 0.0),
+                        texture: asset_server.load("sprites/bananabunch.png"),
+                        ..default()
+                    },
+                    BananaBunch {},
+                ))
+                .insert(BananaPoints { value: 5 });
+        } else {
+            commands
+                .spawn((
+                    SpriteBundle {
+                        transform: Transform::from_xyz(random_x, window.height(), 0.0),
+                        texture: asset_server.load("sprites/banana.png"),
+                        ..default()
+                    },
+                    Banana {},
+                ))
+                .insert(BananaPoints { value: 1 });
+        }
```

Comment:
Maybe we could use the `is_bunch` variable to choose a struct of parameters for the `spawn` and `insert` functions. 

For instance, when `is_bunch` is `true`, we would have a struct containing the sprite path (`sprites/bananabunch.png`), the marker component (`BananaBunch`), and the `value` of the points (5). I'm not really too sure how to actually implement the portion with the marker component in Rust because I'm not that skilled at the language yet... so I'm willing to concede on that if we can't figure out a solution. Possibly generics...?

With this implemented, we would only need to call `spawn` and `insert` one time, which will be important if we want to add more "falling objects" in the future.